cmake/EthPolicy.cmake

```diff
@@ -4,32 +4,16 @@ macro (eth_policy)
 	# link_directories() treats paths relative to the source dir.
 	cmake_policy(SET CMP0015 NEW)
 
-	# let cmake autolink dependencies on windows
-	cmake_policy(SET CMP0020 NEW)
+	# Avoid warnings in CMake 3.0.2:
+	cmake_policy(SET CMP0042 NEW)
+	cmake_policy(SET CMP0043 NEW)
 
-	# CMake 2.8.12 and lower allowed the use of targets and files with double
-	# colons in target_link_libraries,
-	cmake_policy(SET CMP0028 OLD)
+	# allow VERSION argument in project()
+	cmake_policy(SET CMP0048 NEW)
 
-	if (${CMAKE_VERSION} VERSION_GREATER 3.0)
```

Comment:
Is this `>= 3.0`?

Reply:
It should have been `>= 3.0` before, but there is no `VERSION_GREATER_EQUAL`. Moreover, the check was only not to set policies which are not known. The `if (POLICY ...)` is much better protection.

Reply:
So these policies are valid in 3.0.0 then? Makes sense to remove the `if` in that case.

Reply:
All were added in 3.0.0. The check was for compatibility with CMake 2.8.x.